### PR TITLE
Readme review

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +31,7 @@ The guidelines can be found here (TBD)
 
 ## Trade off
```
Comment (janjaap) [inline README.md]: Might as well change this to 'Trade-off' or 'Tradeoff'

File: README.md [hunk 2]
```diff
@@ -51,21 +53,35 @@ The guidelines can be found here (TBD)
 
 ## Usage
 
-##### Development
+### Development
+
+```bash
```
Comment (janjaap) [inline README.md]: `bash` Isn't a supported syntax highlighter. Check https://github.com/github/linguist/blob/master/vendor/README.md for reference.

File: README.md (hunk 1 above)
Comment (aditudorache) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (aditudorache) [reply README.md]: This seem to work in github md. Please see:
https://stackoverflow.com/questions/20303826/highlight-bash-shell-code-in-markdown

File: README.md (hunk 1 above)
Comment (aditudorache) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (janjaap) [reply README.md]: `bash` Isn't recognised as a highlight identifier. If you remove it, you won't see a difference.

**Pull request merged**